.travis.yml

```diff
@@ -1,8 +1,11 @@
 language: ruby
+install:
+  - gem uninstall bundler
+  - gem install bundler --version '1.17.2'
```

Comment:
is there any problems encountered using v2 of bundler?

Reply:
@TayKangSheng  - haven't tried. I'll try shortly.